app/dashboard/router.py

```diff
@@ -124,10 +137,13 @@ def get_queryset(self):
             QuerySet: The Bounty queryset.
 
         """
-        queryset = Bounty.objects.prefetch_related(
-            'fulfillments', 'interested', 'interested__profile') \
-            .current().order_by('-web3_created')
         param_keys = self.request.query_params.keys()
+        queryset = Bounty.objects.prefetch_related(
+            'fulfillments', 'interested', 'interested__profile', 'activities')
+        if 'not_current' not in param_keys:
```

Comment:
curious why were checking for not `not_current` instead of filtering for `current=1` ?  the latter is much more clean IMHO

Reply:
this endpoint by default returns only current bounties, I didn't wanted to change default behavior

Reply:
current bounties, or current activities? 

i think that the endpoint is really only used on this page FWIW.

Reply:
current bounties
it is an api, so I've assumed there is can be some external users
if there is no external users, I can easily reveres flag to more logical one `current=1`

Reply:
> if there is no external users,

there are no external users for this API at this time